packages/wallets/src/features/cfd/components/CompareAccountsCarousel/CompareAccountsCarouselButton.tsx

```diff
@@ -11,22 +10,25 @@ type TPrevNextButtonProps = {
     onClick: () => void;
 };
 
-const CFDCompareAccountsCarouselButton = (props: TPrevNextButtonProps) => {
-    const { enabled, isNext, onClick } = props;
+const CFDCompareAccountsCarouselButton = ({ enabled, isNext, onClick }: TPrevNextButtonProps) => (
+    <IconButton
+        className={classNames('wallets-compare-accounts-carousel-button', {
+            'wallets-compare-accounts-carousel-button--next': isNext,
+            'wallets-compare-accounts-carousel-button--prev': !isNext,
+        })}
+        color='white'
+        disabled={!enabled}
+        icon={
+            isNext ? (
+                <LabelPairedChevronRightLgFillIcon fill='#333333' />
```

Comment:
Worth adding a TODO comment to remove these hard coded hex values when we integrate @deriv-com/ui library to wallets.